google/data_source_dns_managed_zone_test.go

```diff
@@ -68,5 +72,15 @@ resource "google_dns_managed_zone" "foo" {
 data "google_dns_managed_zone" "qa" {
 	name	= "${google_dns_managed_zone.foo.name}"
 }
-`, acctest.RandString(10))
+
+resource "google_dns_managed_zone" "bar" {
+	name		= "bar-zone-%s"
+	dns_name	= "bar.test.com."
```

Comment:
Could we use `tf-test.club` instead of `test.com`? It helps to make sure we don't run into domain validation errors when running the tests.